library/src/main/java/com/novoda/downloadmanager/lib/DefaultNotficationCustomiser.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.downloadmanager.lib;
+
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.app.NotificationCompat;
+
+import com.novoda.downloadmanager.R;
+
+class DefaultNotficationCustomiser implements NotificationCustomiser {
+
+    private final Context context;
+
+    public DefaultNotficationCustomiser(Context context) {
+        this.context = context;
+    }
+
+    @Override
+    public void addActionsForBatch(NotificationCompat.Builder builder, long batchId) {
```

Comment:
Good point! Should we do something or assume that people will be well behaved? :)